tournament/src/api/views/manage_tournament_views.py

```diff
@@ -45,10 +45,12 @@ def patch(request: HttpRequest, tournament_id: int) -> JsonResponse:
 
         try:
             tournament.save()
-            game_created, game_creation_error, game_port = StartTournamentView.create_tournament_game(tournament)
+            game_created, game_creation_error, response = StartTournamentView.create_tournament_game(tournament)
             if not game_created:
-                return JsonResponse({'errors': [game_creation_error]}, status=500)
-            StartTournamentView.send_tournament_start_notification(tournament, game_port)
+                tournament.status = Tournament.CREATED
```

Comment:
if not game_created, the status of the tournament is set to "created"
This is very counterintuitive ..